appdaemon/scheduler.py

```diff
@@ -341,7 +341,7 @@ def is_realtime(self):
 
     def get_next_entries(self):
 
-        next_exec = datetime.datetime.now(pytz.utc).replace(year=3000)
+        next_exec = datetime.datetime.now(pytz.utc).replace(year=3200)
```

Comment:
1. I can't find anything in git commits/blame that  explain  why they set the year to 300
2. Setting to 3200 seems like a bandaid fix - but it does work
3. Would prefer to find a better solution - butt still I approve

Reply:
A better solution to this would be to take the current year and ADD a number divisible by 4 - right now, I feel like on non-leap years thtis will cause issues.

```
next_exec = datetime.datetime.now(pytz.utc)
year_offset = next_exec.year + 3200
next_exec = next_exec + year_offset
```

This would make sure leap or not, it gets the right day.

Reply:
Is there a compelling reason not to use datetime.max?  This is the clearly the intended behavior, from the surrounding code.